internal/serialization/serialization_consts.go

```diff
@@ -100,155 +100,85 @@ const (
+	TypeUUID:                                 "UUID",
+	TypeSimpleEntry:                          "SIMPLE_ENTRY",
+	TypeSimpleImmutableEntry:                 "SIMPLE_IMMUTABLE_ENTRY",
+	TypeJavaClass:                            "JAVA_CLASS",
+	TypeJavaDate:                             "JAVA_DATE",
+	TypeJavaBigInteger:                       "JAVA_BIG_INTEGER",
+	TypeJavaDecimal:                          "JAVA_DECIMAL",
+	TypeJavaArray:                            "JAVA_ARRAY",
+	TypeJavaArrayList:                        "JAVA_ARRAY_LIST",
+	TypeJavaLinkedList:                       "JAVA_LINKED_LIST",
+	TypeJavaDefaultTypeCopyOnWriteArrayList:  "JAVA_COPY_ON_WRITE_ARRAY_LIST",
+	TypeJavaDefaultTypeHashMap:               "JAVA_HASHMAP",
+	TypeJavaDefaultTypeConcurrentSkipListMap: "JAVA_CONCURRENT_SKIP_LIST_MAP",
+	TypeJavaDefaultTypeConcurrentHashMap:     "JAVA_CONCURRENT_HASH_MAP",
+	TypeJavaDefaultTypeLinkedHashMap:         "JAVA_LINKED_HASH_MAP",
+	TypeJavaDefaultTypeTreeMap:               "JAVA_TREE_MAP",
+	TypeJavaDefaultTypeHashSet:               "JAVA_HASH_SET",
+	TypeJavaDefaultTypeTreeSet:               "JAVA_TREE_SET",
+	TypeJavaDefaultTypeLinkedHashSet:         "JAVA_LINKED_HASH_SET",
+	TypeJavaDefaultTypeCopyOnWriteArraySet:   "JAVA_COPY_ON_WRITE_ARRAY_SET",
+	TypeJavaDefaultTypeConcurrentSkipListSet: "JAVA_CONCURRENT_SKIP_LIST_SET",
+	TypeJavaDefaultTypeArrayDeque:            "JAVA_ARRAY_DEQUE",
+	TypeJavaDefaultTypeLinkedBlockingQueue:   "JAVA_LINKED_BLOCKING_QUEUE",
+	TypeJavaDefaultTypeArrayBlockingQueue:    "JAVA_ARRAY_BLOCKING_QUEUE",
+	TypeJavaDefaultTypePriorityBlockingQueue: "JAVA_PRIORITY_BLOCKING_QUEUE",
+	TypeJavaDefaultTypeDelayQueue:            "JAVA_DELAY_QUEUE",
+	TypeJavaDefaultTypeSynchronousQueue:      "JAVA_SYNCHRONOUS_QUEUE",
+	TypeJavaDefaultTypeLinkedTransferQueue:   "JAVA_LINKED_TRANSFER_QUEUE",
+	TypeJavaDefaultTypePriorityQueue:         "JAVA_PRIORITY_QUEUE",
+	TypeJavaDefaultTypeOptional:              "JAVA_OPTIONAL",
+	TypeJavaLocalDate:                        "JAVA_LOCALDATE",
+	TypeJavaLocalTime:                        "JAVA_LOCALTIME",
+	TypeJavaLocalDateTime:                    "JAVA_LOCALDATETIME",
+	TypeJavaOffsetDateTime:                   "JAVA_OFFSETDATETIME",
+	TypeCompact:                              "COMPACT",
+	TypeCompactWithSchema:                    "COMPACT_WITH_SCHEMA",
+	TypeJavaDefaultTypeSerializable:          "JAVA_SERIALIZABLE",
+	TypeJavaDefaultTypeExternalizable:        "JAVA_EXTERNALIZABLE",
+	TypeCsharpCLRSerializationType:           "CSHARP_CLR_SERIALIZATION",
+	TypePythonPickleSerializationType:        "PYTHON_PICKLE_SERIALIZATION",
+	TypeJSONSerialization:                    "JSON",
+	TypeGobSerialization:                     "GO_GOB_SERIALIZATION",
+	TypeHibernate3TypeHibernateCacheKey:      "HIBERNATE3_CACHE_KEY",
+	TypeHibernate3TypeHibernateCacheEntry:    "HIBERNATE3_CACHE_ENTRY",
+	TypeHibernate4TypeHibernateCacheKey:      "HIBERNATE3_CACHE_KEY",
```

Comment:
It could be "HIBERNATE4_CACHE_KEY"

Reply:
Thanks, fixed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/159/commits/ff8e67dd3673dbb32a0005c0509f3b81dc9944a4